app/brave_main_delegate.cc

```diff
@@ -38,7 +38,6 @@
 #include "components/safe_browsing/core/features.h"
 #include "components/security_state/core/features.h"
 #include "components/sync/base/sync_base_switches.h"
-#include "components/sync/driver/sync_driver_switches.h"
```

Comment:
can we remove the dep for this?

Reply:
Same as in https://github.com/brave/brave-core/pull/6195#discussion_r485180830 we rely on `chrome/BUILD.gn` here.